src/pages/derivx/_get-derivx.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import styled from 'styled-components'
+import derivXLogo from '../../images/svg/deriv-x/derivX.svg'
+import derivXQR from '../../images/svg/deriv-x/derivxQR.png'
+import GrayAngle30 from '../../images/common/gray-angle.png'
+import AppleIcon from '../../images/svg/apple-icon.svg'
+import AppGalleryIcon from '../../images/svg/appGallery-icon.svg'
+import AndroidIcon from '../../images/svg/android-icon.svg'
+import BrowserIcon from '../../images/svg/browser-icon.svg'
+import GetAppMobileBG from '../../images/common/dtrader/getAppMobileBG.jpg'
+import CommonHeaderSection from 'components/elements/common-header-section'
+import MultiWidthColumn from 'components/elements/multi-width-column'
+import device from 'themes/device'
+import useBreakpoints from 'components/hooks/use-breakpoints'
+import { useIsRtl } from 'components/hooks/use-isrtl'
+import {
+    derivx_android_url,
+    derivx_huawei_url,
+    derivx_ios_url,
+    derivx_app_url,
+} from 'common/constants'
+import DownloadColumn from 'components/custom/_multi-width-column-download'
+
+const ContentWrapper = styled.div<{ is_rtl: boolean }>`
+    display: flex;
+    gap: 28px;
+    flex: 1;
+    z-index: 2;
+    direction: ${(props) => (props.is_rtl ? 'rtl' : 'ltr')};
+    @media ${device.tabletL} {
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+`
+const TextAndButtonWrapper = styled.div`
+    @media ${device.tabletL} {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+    }
+`
+
+const Wrapper = styled.div`
+    margin-top: 5rem;
+
+    @media ${device.tabletL} {
+        margin-top: -3rem;
+    }
+`
+
+const DerivXGetApp = () => {
+    const { is_mobile_or_tablet } = useBreakpoints()
+    const is_rtl = useIsRtl()
+
+    const items = [
```

Comment:
this doesn't need to be here, it can be outside of the component. can you move it out please?